src/plotting/plot.component.tsx

```diff
@@ -9,15 +9,38 @@ import {
   VictoryLegend,
   VictoryTooltip,
 } from 'victory';
-import { useRecords } from '../api/records';
-import { AxisSettings, PlotType } from '../app.types';
+import {
+  AxisSettings,
+  FullScalarChannelMetadata,
+  PlotType,
+  Record,
+  ScalarChannel,
+} from '../app.types';
+import { format } from 'date-fns';
+
+export const formatTooltipLabel = (
+  label: number,
+  scale: AxisSettings['scale']
+): number | string => {
+  if (scale === 'time') {
+    return format(new Date(label), 'yyyy-MM-dd HH:mm:ss:SSS');
+  }
+  return label;
+};
```

Comment:
This is a nice addition - makes the timestamp tooltips a lot more readable 👍

Reply:
And a necessary one, otherwise they just come out as Unix timestamps